src/app/rundown-view/components/rundown-tooltips/piece-tooltip/piece-tooltip.component.html

```diff
@@ -1,14 +1,27 @@
-<ng-container *ngIf="shouldShowHoverScrub && !isMediaUnavailable">
-  <div #tooltipHoverElement class="c-hover-scrub-hover-element"></div>
+  <div #tooltipHoverElement class="c-tooltip-hover-element"></div>
 
   <sofie-tooltip [tooltipElementHoverMousePosition]="tooltipElementHoverMousePosition">
-    <sofie-video-hover-scrub
-      [filename]="piece.name"
-      [playedDurationInMs]="playedDurationForPartInMs"
-      [tooltipHoverElementWidth]="tooltipHoverElement.clientWidth"
-      [tooltipElementHoverMousePosition]="tooltipElementHoverMousePosition"
-      [durationInMs]="durationInMs"
-      [isJingle]="tv2PieceType === Tv2PieceType.JINGLE">
-    </sofie-video-hover-scrub>
+    <div class="tooltip">
+      <sofie-video-hover-scrub *ngIf="shouldShowHoverScrub && !isMediaUnavailable"
+                               [filename]="piece.name"
+                               [playedDurationInMs]="playedDurationForPartInMs"
+                               [tooltipHoverElementWidth]="tooltipHoverElement.clientWidth"
+                               [tooltipElementHoverMousePosition]="tooltipElementHoverMousePosition"
+                               [durationInMs]="durationInMs"
+                               [isJingle]="tv2PieceType === Tv2PieceType.JINGLE">
+      </sofie-video-hover-scrub>
+
+      <div class="body">
+        <div class='header' [class]="getPieceType().toLocaleLowerCase() + '_color'">
+          <span class="label">{{piece.name}}</span>
+        </div>
+        <div *ngIf="isMediaUnavailable" class="source-unavailable-banner">
+          <sofie-icon-button [iconButtonSize]="IconButtonSize.M" [iconButton]="IconButton.TRIANGLE_EXCLAMATION">
+          </sofie-icon-button>
+          {{sourceUnavailableLabel}}
+        </div>
+          <ng-content></ng-content>
```

Comment:
As mentioned in my other comments, you shouldn't content-inject into this component.

Reply:
No longer happens!